org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationRegistry.java

```diff
@@ -339,7 +342,9 @@ protected void putInternal(UiNotificationMessageDo message, boolean publishOverC
     finally {
       m_lock.writeLock().unlock();
     }
-    if (publishOverCluster) {
+    if (Optional.ofNullable(options)
```

Comment:
Please use same pattern as in other methods

    if (options == null) {
      options = new UiNotificationPutOptions();
    }